docs/use_cases/model_applications/short_range/MODEMultivar_fcstHRRR_obsMRMS_HRRRanl.py

```diff
@@ -0,0 +1,206 @@
+# time on the super-object-masked field(s) to output attribute statistics for
+# the field(s).
+#
+# **Note:** The second MODE run can also be run directly on the super objects if
+# field-specific statistics, such as intensity, is not desired.
+# 
+
+##############################################################################
+# METplus Workflow
+# ----------------
+#
+# The following tools are used for each run time:
+#
+# MODE(mv), GenVxMask(fcst_super), GenVxMask(obs_super), MODE (super)
+# 
+# Where the first instance of MODE runs over multiple variables to identify
+# super objects for the forecast and observation, GenVxMask masks the raw input
+# field(s) using the super objects, and the second instance of MODE is run
+# traditionally to compare the masked forecast and observed super objects and
+# and provide statistics.
+#
+# This example runs a single forecast hour.
+#
+# | **Initialization:** 2021020100
+# | **Forecast lead:** 21
+#
+
+##############################################################################
+# METplus Configuration
+# ---------------------
+#
+# METplus first loads all of the configuration files found in parm/metplus_config,
+# then it loads any configuration files passed to METplus via the command line
+# with the -c option, i.e. -c parm/use_cases/model_applications/short_range/MODEMultivar_fcstHRRR_obsMRMS_HRRRanl.conf
+#
+# .. highlight:: bash
+# .. literalinclude:: ../../../../parm/use_cases/model_applications/short_range/MODEMultivar_fcstHRRR_obsMRMS_HRRRanl.conf
+
+##############################################################################
+# MET Configuration
+# ---------------------
+#
+# METplus sets environment variables based on user settings in the METplus configuration file. 
+# See :ref:`How METplus controls MET config file settings<metplus-control-met>` for more details. 
+#
+# **YOU SHOULD NOT SET ANY OF THESE ENVIRONMENT VARIABLES YOURSELF! THEY WILL BE OVERWRITTEN BY METPLUS WHEN IT CALLS THE MET TOOLS!**
+#
+# If there is a setting in the MET configuration file that is currently not supported by METplus you'd like to control, please refer to:
+# :ref:`Overriding Unsupported MET config file settings<met-config-overrides>`
+#
+# .. note:: See the :ref:`MODE MET Configuration<mode-met-conf>` section of the User's Guide for more information on the environment variables used in the file below:
+#
+# .. highlight:: bash
+# .. literalinclude:: ../../../../parm/met_config/MODEConfig_wrapped
+
+##############################################################################
+# Running METplus
+# ---------------
+#
+# This use case can be run two ways:
+#
+# 1) Passing in MODEMultivar_fcstHRRR_obsMRMS_HRRRanl.conf then a user-specific system configuration file::
+#
+#        run_metplus.py -c /path/to/METplus/parm/use_cases/model_applications/short_range/MODEMultivar_fcstHRRR_obsMRMS_HRRRanl.conf -c /path/to/user_system.conf
```

Comment:
The documentation for this is slightly outdated. A user no longer needs the "-c" between each configuration file. These should be removed here, as well as the second method of running the use case (just below this line).